src/streamer.rs

```diff
@@ -1107,4 +1131,17 @@ mod test {
             WINDOW_SIZE + 9
         );
     }
+
+    #[test]
+    pub fn validate_blob_against_window_test() {
+        let recvd: u64 = 100;
+        assert!(validate_blob_against_window(0, 90 + WINDOW_SIZE, 90, recvd).is_none());
```

Comment:
Can you use `assert_eq!` in all of these? It'll give you better error messages.

Reply:
Sure. I'll change it.